drivers/shared/structs/structs.go

```diff
@@ -0,0 +1,10 @@
+package structs
+
+import "errors"
+
+// CheckBufSize is the size of the buffer that is used for job output
+const CheckBufSize = 4 * 1024
+
+// DriverStatsNotImplemented is the error to be returned if a driver doesn't
+// implement stats.
+var DriverStatsNotImplemented = errors.New("stats not implemented for driver")
```

Comment:
Wow, are these really the only things in this package? Could they be moved to `client/structs` with causing import cycles?

Reply:
Keeping CheckBufSize in `plugins/drivers/driver.go` and moving this error there seems reasonable as well. That package is already sort of the shared driver structs package.

Reply:
The problem with putting them in `plugins/drivers` is that they're required by the executor implementation so you end up with:

```
package github.com/hashicorp/nomad
        imports github.com/hashicorp/nomad/command
        imports github.com/hashicorp/nomad/client/allocrunner/taskrunner/restarts
        imports github.com/hashicorp/nomad/plugins/drivers
        imports github.com/hashicorp/nomad/plugins/drivers/utils
        imports github.com/hashicorp/nomad/drivers/shared/executor
        imports github.com/hashicorp/nomad/plugins/drivers
```

Reply:
They merge cleanly into client/structs tho 👍